### DISCUSSION TITLE
IOS CRASH : use render renderTarget

### DISCUSSION THREAD

**Discussion by Tony-qty** [google/filament#4873] [Q&A]
I am trying to replace swapchain from ios layer to render target. But  there is a crash when I start render
![image](https://user-images.githubusercontent.com/7675144/142411111-fd745732-722c-4ece-a81a-2471bc667d52.png)
![image](https://user-images.githubusercontent.com/7675144/142411135-0b855f84-4834-4bb5-bcfe-86a99ebbc976.png)
so ,what should I do

**Comment by simonsland** [comment]
Have you solve this problem? Recently I'm trying to render with renderTarget without sample code to show how to use renderTarget. I've also met this problem too.
